src/main/java/com/google/sps/servlets/SEEYouGuiceServletContextListener.java

```diff
@@ -0,0 +1,14 @@
+package com.google.sps.servlets;
+
+import com.google.inject.servlet.ServletModule;
+import com.google.inject.servlet.GuiceServletContextListener;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+
+public class SEEYouGuiceServletContextListener extends GuiceServletContextListener {
```

Comment:
Try to stick to camelcase `SeeYouGuiceServletContextListener` also in the file name